### PR TITLE
fix list runtime 500 error

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind bug

#### What this PR does / why we need it:
find creating runtime by release should filter yaml name 

#### Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://terminus-org.app.terminus.io/erda/workBench/projects/387/issues/all?id=200737&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDQzNiJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=429&type=BUG)

#### Specified Reviewers:

/assign @Effet

### CODE REVIEW

File: modules/orchestrator/utils/pipeline_utils.go [hunk 1]
```diff
@@ -164,39 +164,40 @@ func FindCreatingRuntimesByRelease(appID uint64, envs map[string][]string, ymlNa
...
-		// first condition means user have permission
-		// second condition means the pipeline is used to deploy runtime by release
-		// third condition means that the runtime data of db has higher priority
-		// And one branch corresponds to only one rutime
-		if ok && strings.Contains(v.YmlName, "dice-deploy-release") && !strutil.Exist(runtimeBranchs, branch) {
-			// get pipeline detail to confirm whether the runtime has been created
-			piplineDetail, err := bdl.GetPipeline(v.ID)
-			if err != nil {
-				return nil, err
+		if strings.Contains(v.YmlName, "dice-deploy-release") {
```
Comment (Effet) [inline modules/orchestrator/utils/pipeline_utils.go:167]: if indent too deep, you should
```go
if !strings.Contains() {
  continue
}
// other codes
```

File: modules/orchestrator/utils/pipeline_utils.go [hunk 2]
```diff
@@ -164,39 +164,40 @@ func FindCreatingRuntimesByRelease(appID uint64, envs map[string][]string, ymlNa
...
-					},
-					LastOperateTime: *v.TimeBegin,
-					LastOperator:    fmt.Sprintf("%v", v.Extra.RunUser.ID),
-				})
+			branch := branchSlice[3]
+			runtimeBranchs, ok := envs[strings.ToLower(v.Extra.DiceWorkspace)]
+			// first condition means user have permission
+			// second condition means that the runtime data of db has higher priority
+			// And one branch corresponds to only one rutime
+			if ok && !strutil.Exist(runtimeBranchs, branch) {
```
Comment (Effet) [inline modules/orchestrator/utils/pipeline_utils.go:177]: same, fast fail

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/orchestrator/utils/pipeline_utils.go (hunk 1 above)
Comment (QvodSoldier) [reply modules/orchestrator/utils/pipeline_utils.go]: done

File: modules/orchestrator/utils/pipeline_utils.go (hunk 2 above)
Comment (QvodSoldier) [reply modules/orchestrator/utils/pipeline_utils.go]: done

**Review by Effet — APPROVED** [review]

**Pull request merged**